distribution/lib/Standard/Base/0.0.0-dev/src/Data/List.enso

```diff
@@ -404,10 +404,10 @@ type List
 
     to_text : Text
     to_text self =
-        go l t = case l of
-            Nil -> t + "Nil"
-            Cons x xs -> @Tail_Call go xs (t + "Cons " + x.to_text + " ")
-        go self ""
+        go l t e = case l of
```

Comment:
This is unrelated to [pivotal 183578531](https://www.pivotaltracker.com/n/projects/2539304/stories/182522644) but fixes a problem introduced by #4023  I realized during Monday's demo meeting.